modtools/create-unit.lua

```diff
@@ -843,7 +986,8 @@ local validArgs = utils.invert({
   'quantity',
   'duration',
   'locationRange',
-  'locationType'
+  'locationType',
+  'equip'
```

Comment:
Not important, but using trailing commas (in this case, after `'equip'`) can clean up the git history a bit. As an example, because of the comma you added to the `locationType` line, someone looking to find when that was added with `git blame` would end up here first.
(Admittedly, we don't really have any style guidelines when it comes to this yet, but I thought I'd mention it.)

Reply:
Makes sense; I'll edit this one and keep it in mind for any future scripts. I'm surprised I hadn't seen this mentioned before; most of the scripts I've come across in this repo seem to lack trailing commas in this context.